engines/advancedDetector.cpp

```diff
@@ -566,7 +566,7 @@ static bool getFilePropertiesIntern(uint md5Bytes, const AdvancedMetaEngine::Fil
 		if (fileProps.size != 0)
 			return true;
 
-		Common::SeekableReadStream *dataFork = macResMan.getDataFork();
+		Common::SeekableReadStream *dataFork = Common::MacResManager::openFileOrDataFork(fname, fileMapArchive);
```

Comment:
Same question. Does it mean that when a file is plain and could be missing the required resource fork, we will happily report it as data fork?

E.g. in this particular case we need to check explicitly that the data _fork_ is present

Reply:
Following check for non-null is sufficient. This doesn't change from getDataFork